src/workloads/sharding/ReshardCollection.yml

```diff
@@ -44,6 +43,16 @@ GlobalDefaults:
       OperationCommand:
         Filter: {_id: {^RandomInt: {min: 1, max: *DocumentCount}}}
         Update: {$inc: {counter: 1}}
+    - OperationName: insertOne
+      OperationCommand:
+        Document:
+          _id: {^RandomInt: {min: *DocumentCount, max: 9000000}}
```

Comment:
I think we should define a constant under `GlobalDefaults` for the min and max, e.g. name it `DocumentIdMinForInserts` and `DocumentIdMaxForInserts`? and add a comment to explain why that is the range to use, e.g. by pointing out that the `MonotonicSingleLoader` used by the `LoadInitialData` phase generates documents with monotonically increasing `_id`, starting from `_id: 0`. Same for the other workloads.

[question] Why is it 900'000 instead of some round number like 1'000'000?